cmd/agent/main.go

```diff
@@ -27,15 +40,75 @@ func main() {
 				return fmt.Errorf("another disruptor command is already in execution")
 			}
 
+			// cpu profiling
+			if cpuProfile {
+				var profileFile *os.File
+				profileFile, err = os.Create(cpuProfileFileName)
+				if err != nil {
+					return fmt.Errorf("error creating CPU profiling file %s: %w", cpuProfileFileName, err)
```

Comment:
How would you reword the message?

Reply:
The most common way I see is simply removing the "error" word:

```
return fmt.Errorf("creating CPU profiling file %q: %w", cpuProfileFileName, err)
```

Rationale being that otherwise you can end up with a log like:

```
[ERROR]: error initializing this: error creating file: error creating folder
```

As opposed to:

```
[ERROR]: initializing this: creating filed: creating folder
```

Reply:
Makes sense. However, if there is no log then the message looks incomplete. What I've done in other parts of the code is  use different wording such as "failed to do something"